source/api_reference/index.html.md.erb

```diff
@@ -5,116 +5,131 @@ weight: 60
-
-![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/pay-transaction-states.png)
-
-You can check the status of a given payment using the <a href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id" target="blank">Find payment by ID</a> API call (link opens in new window).
-
-The response will include a ``status`` value as described in the table below, and a true/false ``finished`` value which indicates if the status can change.
-
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-| status value | Meaning                                                                                        | finished value |
-|--------------|------------------------------------------------------------------------------------------------|----------------|
-| created      | Payment created; user has not yet visited next_url                                             | false          |
-| started      | User has visited next_url and is entering payment details                                      | false          |
-| submitted    | User has submitted payment details but has not yet clicked Confirm                             | false          |
-| success      | User successfully completed the payment                                                        | true           |
-| failed       | User attempted to make a payment but the payment did not complete. Possible reasons: user cancelled; user entered payment details but never confirmed; payment declined due to insufficent funds; payment declined due to fraud or other checks | true           |
-| cancelled    | Your service cancelled the payment using an API call or the self-service site.                 | true           |
-| error        | Something went wrong with GOV.UK Pay or the underlying Payment Service Provider.               | true           |
+The following diagram gives an overview of the payment status lifecycle and the
+possible outcomes, excluding for delayed capture payments:
+
+![](images/payment-states.svg)
+
+You can check the status of a payment using the <a
+href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id"
+target="blank">Find payment by ID</a> API call [external link].
+
+A successful response includes ``status`` and ``finished`` values:
+
+<table style="width:100%">
+  <tr>
+  <th><code>status</code></th>
+  <th>Meaning</th> 
+  <th nowrap><code>finished</code></th>
+  </tr>
+  <tr>
+    <td> created </td>     
+    <td> Payment created. User has not yet visited <code>next_url</code> </td>
+    <td> false </td> 
+  </tr>
+  <tr>
+    <td> started </td>
+    <td> User has visited <code>next_url</code> and is entering payment details </td> 
+    <td> false </td>
+  </tr>
+  <tr>
+    <td> submitted </td>
+    <td> User has submitted payment details but has not yet selected 'Confirm' </td> 
```

Comment:
This can also be if the user has pressed the confirm button and the payment is a delayed capture one